### PR TITLE
Update fmv.rst

### PR DESCRIPTION
**Pull request by pixelgeek**
Fix FFT typo in section header

### CODE REVIEW

File: source/clear-linux/tutorials/fmv.rst [hunk 1]
```diff
@@ -186,7 +186,7 @@ verify, enter the following commands:
     vpaddd (%r8,%rax,1),%ymm0,%ymm0
     vmovdqu %ymm0,(%rcx,%rax,1)
 
-FTT project example
+FFT project example
```
Comment (mvincerx) [inline source/clear-linux/tutorials/fmv.rst]: Thanks for correcting the typo.  The example below this shows use of FFTW, not FFT. I'd like clarification on this.  
http://www.fftw.org/faq/section1.html#whatisfftw 
@pixelgeek , @phmccarty, @mrkz can one of you help determine which is correct here, FFT or FFTW?

Comment (phmccarty) [reply source/clear-linux/tutorials/fmv.rst]: FFTW is the project name, and FFT is an algorithm, so I would replace all mention of FFT in this tutorial with FFTW.

Comment (mvincerx) [reply source/clear-linux/tutorials/fmv.rst]: Hello @pixelgeek. Will you please make edits per comments from @phmccarty?

Comment (pixelgeek) [reply source/clear-linux/tutorials/fmv.rst]: Updated - some are generic FFT references, some are FFTW library code references.

**Comment by mvincerx** [comment]
LGTM

**Pull request merged**